src/index.js

```diff
@@ -48,7 +48,13 @@ export default function supernova(env) {
       const [opts] = useState(options);
       const selectionsAPI = useSelections();
       const constraints = useConstraints();
-      const [selectionsAndTransform] = useState({ api: selectionsAPI, setState: setSelectionState, linked: false, transform: {} });
+      const [selectionsAndTransform] = useState({
```

Comment:
Yes we would probably have to rename this one again! Any suggestions are welcome of course

Reply:
selectionAndTransformAndConstraints

Reply:
Can't think of anything on top of my head. Maybe something to indicate that is persisted and stuff can be update updated without reseting the entire object.